lua/fspectate/cl_init.lua

```diff
@@ -512,6 +514,22 @@ local function drawHelp()
     draw.WordBox( 2, bottomLeft.x, bottomLeft.y + 12, "Left click to spectate!", "UiBold", uiBackground, uiForeground )
 end
 
+--[[---------------------------------------------------------------------------
+Draws a crosshair
+---------------------------------------------------------------------------]]
+local function drawCrosshair()
+    if not showCrosshair then return end
+    local h = ScrH() / 2
+    local w = ScrW() / 2 - 1 -- -1 to make it exactly match with the hl2 crosshair.
+
+    surface.SetDrawColor( 0, 0, 0, 255 )
+    surface.DrawLine( w + 1, h + 10, w + 1, h - 10 )
+    surface.DrawLine( w + 10, h + 1, w - 10, h + 1 )
+    surface.SetDrawColor( 255, 255, 255, 255 )
```

Comment:
would personally prefer green with black outline.

Reply:
Was planning a follow up PR with even more customizability, could add a custom color option if you want. If you make a quick suggestion for the color change i'll add that.